libr/reg/arena.c

```diff
@@ -43,7 +43,11 @@ R_API ut8 *r_reg_get_bytes(RReg *reg, int type, int *size) {
 		*size = sz;
 	}
 	buf = malloc (sz);
-	if (buf && reg->regset[type].arena->bytes) { // TODO change 2nd check to something more reasonable
+	if (!reg->regset[type].arena->bytes) { // DBG
+		eprintf ("sz %d\n", sz);
+		*(ut8 *)0 = 0;
```

Comment:
use r_sys_breakpoint() for debugging :P

Reply:
or just use an assert

Reply:
i'll use r_sys_breakpoint()/an assert after this pr